### PR TITLE
SQL-1888: Fix leaking garasign password, local passwords are fine

### PR DESCRIPTION
**Pull request by pmeredit**

### CODE REVIEW

**Review by mattChiaravalloti — APPROVED** [review]

**Comment by mattChiaravalloti** [comment]
Since mine is just a quick-win ticket and this one looks like it'll be in soon, I'll just hold off until this is merged and then I'll update #179 .

**Review by bucaojit — APPROVED** [review]
LGTM

**Pull request merged**
